smoker/client/cli.py

```diff
@@ -35,6 +36,105 @@
     'gray' : '\033[0;37m',
 }
 
+CONFIG_FILE = '/etc/smokercli.yaml'
```

Comment:
What's the use case for this? You can only put a valid python module path to the config file, which means user either has to have root to install it and therefore modify the config file in etc. Otherwise a the smokercli has to be run with custom python path which sounds quite like a pretty rare use case.